scripts/exwave_prep.sh

```diff
@@ -104,6 +104,14 @@
   ymdh_rst_ini=`$NDATE ${RSTOFFSET} $YMDH`
   DT2RSTH=$(( DT_2_RST_WAV / 3600 ))
   RST2OFFSET=$(( ${DT2RSTH} + ${RST2IOFF_WAV} ))
+  if [ ${RST2OFFSET} -lt ${WAVHCYC} ]; then
+    echo ' '
+    echo " FATAL ERROR: RST2 STRIDE SMALLER THAN RUN CYCLE "
```

Comment:
Message indicates a fatal error but does not exit

Reply:
so should I use this version to do the parallel test tomorrow?

Reply:
Exit on error [included](https://github.com/WalterKolczynski-NOAA/global-workflow/pull/8/commits/8fcbfc4d393e19f22dea5734e086eaab37f4af11#diff-6070fe56330029b56168ba65e9e5ac69R111-R112)